site/en/posts/chromeos-112-113-release-notes.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'ChromeOS 112 and 113 improve Screen Capture, screensavers, and security'
+metadesc: 'The latest ChromeOS release includes updates to Screen Capture, screensavers, and security.'
+tags:
+  - announcement
+  - product news
+  - technical
+authors:
+  - samrichard
+date: 2023-05-06
+---
+
+ChromeOS 112 and 113 brought an array of security improvements and productivity enhancements—ranging from easy access to the Rupee symbol to the introduction of the Rupee symbol to instant administrative reboots.
+
+## ChromeOS 112: Rupee symbol and Screen Capture updates
+
+ChromeOS 112 released on April 6, 2023, introducing several user productivity features and application updates. Two highlights:
```

Comment:
Instead of highlights for 112 and 113, can we style these like the other items, where we call out what was released directly?